docs/TMC_Drivers.md

```diff
@@ -70,124 +62,259 @@ of the motor, so the TMC cannot detect stalls anymore. It is advised
+center). It should be possible to find the highest sensitivity that
+homes reliably (settings with higher sensitivity result in small or no
+movement). Note the found value as `maximum_sensitivity`. (If the
+minimum possible sensitivity (SGTHRS=0 or SGT=63) is obtained without
+any carriage movement then something is not correct with the
+diag/sg_tst pin wiring or configuration and it must be corrected
+before continuing.)
+
+When searching for maximum_sensitivity, it may be convenient to jump
+to different VALUE settings (so as to bisect the VALUE parameter). If
+doing this then be prepared to issue an `M112` command to halt the
+printer, as a setting with a very low sensitivity may cause the axis
+to repeatedly "bang" into the end of the rail.
+
+Be sure to wait a couple of seconds between each homing attempt. After
+the TMC driver detects a stall it may take a little time for it to
+clear its internal indicator and be capable of detecting another
+stall.
+
+During these tuning tests, if a `G28 X0` command does not move all the
+way to the axis limit, then be careful with issuing any regular
+movement commands (eg, `G1`). Klipper will not have a correct
+understanding of the carriage position and a move command may cause
+undesirable and confusing results.
+
+### Find lowest sensitivity that homes with one touch
+
+When homing with the found `maximum_sensitivity` value, the axis
+should move to the end of the rail and stop with a "single touch" -
+that is, there should not be a "clicking" or "banging" sound. (If
+there is a banging or clicking sound at maximum_sensitivity then the
+homing_speed may be too low, the driver current may be too low, or
+sensorless homing may not be a good choice for the axis.)
+
+The next step is to again continually move the carriage to a position
+near the center of the rail, decrease the sensitivity, and run the
+`SET_TMC_FIELD` `G28 X0` commands - the goal is now to find the lowest
+sensitivity that still results in the carriage successfully homing
+with a "single touch". That is, it does not "bang" or "click" when
+contacting the end of the rail. Note the found value as
+`minimum_sensitivity`.
+
+### Update printer.cfg with sensitivity value
+
+After finding `maximum_sensitivity` and `minimum_sensitivity`, use a
+calculator to obtain the recommend sensitivity as
+`minimum_sensitivity + (maximum_sensitivity - minimum_sensitivity)/3`.
+The recommended sensitivity should be in the range between the minimum
+and maximum, but slightly closer to the minimum. Round the final value
+to the nearest integer value.
+
+For tmc2209 set this in the config as `driver_SGTHRS`, for other TMC
+drivers set this in the config as `driver_SGT`.
+
+If the range between `maximum_sensitivity` and `minimum_sensitivity`
+is small (eg, less than 10) then it may result in unstable homing. A
+faster homing speed may increase the range and make the operation more
+stable.
```

Comment:
I don't know if less than 10 range being equal to unstable homing is a safe statement. Having gone through a few iterations of tests I am seeing my range be from 1-4 on both x and y with good results as far as I can tell without having run prints.

This sort of competes with the guidance above about wanting a slower speed to reduce load on the frame, but if we are to try getting a wider range, I would guess we would need higher speeds. Would it be worth making an acknowledgement to that?

I ended up settling with 40 for the speed and keeping my max as the threshold, since for the most part my motor and driver pairing seems to work great or not at all.